docs/development.md

```diff
@@ -343,7 +343,17 @@ To run E2E locally, set `AZURE_CLIENT_ID`, `AZURE_CLIENT_SECRET`, `AZURE_SUBSCRI
 ./scripts/ci-e2e.sh
 ```
 
-You can optionally set `AZURE_SSH_PUBLIC_KEY_FILE` to use your own SSH key.
+You can optionally set the following variables:
+
+| Variable                           | Description                                                                                                               | Default                                                                                         |
+|------------------------------------|---------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------|
+| `AZURE_SSH_PUBLIC_KEY_FILE`        | Use your own SSH key.                                                                                                     | `${GOPATH}/src/sigs.k8s.io/cluster-api-provider-azure/.sshkey`                                  |
+| `AZURE_CONTROL_PLANE_MACHINE_TYPE` | The machine type of the control plane.                                                                                    | `Standard_D2s_v3`                                                                               |
+| `AZURE_NODE_MACHINE_TYPE`          | The machine type of the node machine.                                                                                     | `Standard_D2s_v3`                                                                               |
+| `E2E_CONF_FILE`                    | The path of the [E2E configuration file](https://cluster-api.sigs.k8s.io/developer/e2e.html#defining-an-e2e-config-file).      | `${GOPATH}/src/sigs.k8s.io/cluster-api-provider-azure/test/e2e/config/azure-dev.yaml`            |
+| `E2E_ARTIFACTS`                    | The path of the artifacts folder.                                                                                         | `${GOPATH}/src/sigs.k8s.io/cluster-api-provider-azure/_artifacts`                               |
```

Comment:
Isn't `ARTIFACTS` a predefined prow variable? if so we shouldn't use a different name for it so that CI jobs can pick up the prow value

Reply:
Yea it's a predefined variable in Prow. Let me modify this.